assets/js/state/settings.js

```diff
@@ -1,29 +1,20 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  eulaVisible: false,
   isPremium: false,
 };
 
 export const settingsSlice = createSlice({
   name: 'settings',
   initialState,
   reducers: {
-    setEulaVisible: (state) => {
-      state.eulaVisible = true;
-    },
     setIsPremium: (state) => {
       state.isPremium = true;
     },
-    acceptEula: (state) => {
-      state.eulaVisible = false;
-    },
   },
 });
 
-export const ACCEPT_EULA = 'ACCEPT_EULA';
-
-export const { setEulaVisible, setIsPremium, acceptEula } =
+export const { setIsPremium  } =
```

Comment:
As per the previous comment this might not be needed anymore.

Reply:
I think it is. Being premium or not was not only about the eula.
We still have this `Premium: true/false` thing, based on the artifacts creation and if there are sles subscriptions active

Reply:
you're right :+1: